plugins/github-deployments/src/mocks/mocks.ts

```diff
@@ -0,0 +1,67 @@
+import { QueryResponse } from '../api';
```

Comment:
this should be below the copyright header :)

Reply:
Auto import :(